constraint/pkg/regorewriter/regorewriter_test.go

```diff
@@ -495,3 +499,56 @@ func TestRegoRewriterErrorCases(t *testing.T) {
 		}
 	}
 }
+
+func TestRegoRewriterParseErrors(t *testing.T) {
+	testcases := []RegoRewriterTestcase{
+		{
+			name:      "fail to add base",
+			baseSrcs:  map[string]string{"invalid.rego": RegoSrc("foo", Body("something invalid"))},
+			wantError: true,
+		},
+		{
+			name:      "fail to add lib",
+			libSrcs:   map[string]string{"invalid.rego": RegoSrc("foo", Body("something invalid"))},
+			wantError: true,
+		},
+	}
+	for _, tc := range testcases {
+		t.Run(tc.name, tc.Run)
+	}
+}
+
+func TestRegoRewriterAddLibFromFs(t *testing.T) {
+	tests := []struct {
+		name    string
+		path    string
+		wantErr bool
+	}{
+		{
+			name:    "path not exist",
+			path:    "foo/bar",
+			wantErr: true,
+		},
+		{
+			name: "path without test",
+			path: "./",
+		},
+	}
+	for _, tt := range tests {
```

Comment:
Nit: prefer "tc" instead of "tt" for test cases. Feel free to disregard this comment.